docs/tutorial/part-one/index.md

```diff
@@ -409,7 +409,7 @@ case) and you'll see your newly published site! Great work!
 
 [Gatsby Cloud](https://gatsbyjs.com) is a platform built specifically for Gatsby sites, with features like real-time previews, fast builds, and integrations with dozens of other tools. It's the best place to build and deploy sites built with Gatsby, and you can use Gatsby Cloud free for personal projects.
 
-To deploy your site to Gatsby Cloud, create an account on [GitHub](https://github.com) if you don't have one. GitHub allows you to host and collaborate on code projects using Git for version control.
+To deploy your site to Gatsby Cloud, create an account on [GitHub](https://github.com) if you don't have one. GitHub allows you to host and collaborate on code projects using Git for version control. You will need to [generate a new SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and [add the SSH key to your Github account](https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account), if you have not done so previously.
```

Comment:
Is this a required step? I've been able to use GitHub on my laptop without setting up an SSH key. Are there certain requirements that would force someone to use an SSH key (like device, repo settings, etc.)?

Reply:
I believe you either have to authenticate with your username and password, or use an ssh key.